en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+# Develop an API Proxy from Git Repository
+
+An API proxy acts as an intermediary between an existing API and Choreo, intercepting all requests made to the API. It also functions as a managed API, allowing you to apply essential API management features such as security policies and rate limiting.
+
+In this guide, you will:
+
+- Create an API proxy component to expose an existing API.
+- Deploy the API proxy.
+- Test the API proxy to verify its functionality.
+- Manage the API.
+- Consume the API.
+
+## Prerequisites
+
+- If you're signing in to the Choreo Console for the first time, create an organization:
+    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in using your preferred method.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use.
+    4. Click **Create**.
+
+    This creates the organization and opens the **Project Home** page of the default project created for you.
+
```

Comment:
Since we are using a sample from the Choreo samples repo in this guide, we would need to ask users to  fork the repo and mention that the repo will have the Proxy from GitHub sample.

Reply:
what would the better way to do that ? an info banner asking to fork the sample repo ..

Reply:
No need to fork the repo , right. Anyone can use the repo without forking it. But when they need to change the exiting source , then only they need to fork it.

Reply:
Mention the step as a prerequisite